integration-tests/src/test/java/oracle/kubernetes/operator/utils/TestUtils.java

```diff
@@ -939,6 +942,60 @@ public static void verifyAfterDeletion(Domain domain) throws Exception {
     k8sTestUtils.verifyNoClusterRoleBindings(domain1LabelSelector);
   }
 
+  /**
+   * create oracle db pod in the k8s cluster
+   *
+   * @param dbPropsFile - db properties file
+   * @return - OracleDB instance
+   * @throws Exception - if any error occurs when creating Oracle DB pod
+   */
+  public static OracleDB createOracleDB(String dbPropsFile) throws Exception {
+    OracleDB oracledb = new OracleDB(dbPropsFile);
+
+    // check the db is ready
+    String dbnamespace = oracledb.getNamespace();
+
+    String cmd = "kubectl get pod -n " + dbnamespace + " -o jsonpath=\"{.items[0].metadata.name}\"";
+    ExecResult result = ExecCommand.exec(cmd);
+    String podName = result.stdout();
+
+    logger.info("DEBUG: db namespace=" + dbnamespace);
+    logger.info("DEBUG: podname=" + podName);
+    TestUtils.checkPodReady("", dbnamespace);
+
+    // check the db is ready to use
+    cmd = "kubectl logs " + podName + " -n " + dbnamespace;
+    logger.info("Running command: " + cmd);
+    result = ExecCommand.exec(cmd);
+    int timeout = 15;
+    int i = 0;
+    while ((i < timeout) && !result.stdout().contains("The database is ready for use")) {
```

Comment:
If you want you can call BaseTest.getMaxIterationsPod() and BaseTest.getWaitTimePod() which uses values from src/test/resources/OperatorIT.properties.